challenge1.js

```diff
@@ -0,0 +1,22 @@
+// Print all the prime numbers between 0 and 100.
+
+function isPrime(n) {
+    if (n <= 1) {
+      return false;
+    }
+    
+    const sqrtNumber = Math.floor(Math.sqrt(n));
+    for (let i = 2; i <= sqrtNumber ; i++) {
+        if (n % i === 0) {
+            return false;
+        }
+    }
+    
+    return true;
+}
+for (let i = 2; i <= 100; i++) {
```

Comment:
What is the purpose of getting `n` from the user then?

Reply:
n is not being taken as input from the user . It is being used to check whether a given number is prime or not.